packages/bits/spec/components/toolbar/toolbar.e2e.ts

```diff
@@ -103,10 +103,10 @@ describe("USERCONTROL toolbar: ", () => {
 
         it("should toolbar resized and hide all items in menu", async () => {
             const width = "100";
-            await toolbar.popup.open();
             const toolbarItems = await toolbar.getAllVisibleItemsCount();
             await resizableToolbarTextboxAtom.acceptText(width);
             await browser.wait( async () => toolbarItems !== 0, 500);
+            await toolbar.popup.open();
```

Comment:
what was the reason this needed to be moved?

Reply:
on resize trigger menu now lose focus. On this event we close menu and it isn't possible to reach menu items in visual test

Reply:
okay, thanks for the explanation